### DISCUSSION TITLE
visualize my skills

### DISCUSSION THREAD

**Discussion by Andreexp** [community/community#123377] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi,
I have several skills and I am looking for a repo or tool or code to visualize them in my GitHub intro page. (Uniformed Icons)
Thank you in advance for your suggestions.

**Comment by sajadh76** [comment, marked as answer]
Hello,

Take a look at the following one:  [Link](https://github.com/LelouchFR/skill-icons)

**Reply by Andreexp** [reply]
Thanks! Exactly the one I was looking for

**Reply by ebndev** [reply]
@sajadh76 thanks for sharing this 😄

**Reply by LelouchFR** [reply]
Thank you :) 🙏

**Comment by GustavoQuispeC** [comment]
Maybe this can help you.  https://simpleicons.org/
